docs/blog/newsletter-november-2024.rst

```diff
@@ -0,0 +1,73 @@
+
+A hiring manager focused on how quickly the person can get up to speed with the team. If you’re a quick learner, have experience with other relevant tools, and are self-motivated enough to self-study (if necessary), lack of specific tooling experience isn’t a problem. If you don’t have experience with a specific tool, use an interview to relate how you learned other tools.
+
+Several concerns included:
+
+* **If your resume doesn’t mention ANY tool specified in a job posting, the application may get rejected automatically.** This is particularly true if you’re going through a recruiter. It may be worthwhile to gain at least a novice-level exposure to include on your resume. (Don’t lie about your experience.) Many tools are either free or have free trials available.
+* **There may be some basic tool-related knowledge that’s expected with certain types of documentation.** If you’re transitioning to another domain, consider getting at least minimal exposure to relevant knowledge (such as Markdown, Git, or HTML). 
+* **Some "tools" are actually concepts.** You may want to research what certain items (such as DITA, distributed systems, or version control) involve so you can discuss them.
+
+Some companies do expect to train new hires. If nothing else, new hires need to learn the way the team uses a particular tool. If hired, be prepared to learn from others even if you have experience with their tools.
+
+----------------------------------------------------
+Dealing with images and other assets in docs-as-code
+----------------------------------------------------
+
+A couple of recent questions regarding `#docs-as-code <https://writethedocs.slack.com/archives/C72NZ18FR>`__ touched on how to handle binary files such as images in a docs-as-code setup, which is primarily built to track text files. One concern was the size of the binary files, which can make a Git repository balloon up to gigabytes in size. Another issue was whether to store images next to the file they're used in or in a separate directory only for images.
+
+Unsurprisingly, the answers on what to do varied based on what was required. For example, if you only need simply diagrams, you can create them using a diagrams-as-code tool such as `Mermaid <https://mermaid.js.org/>`__ or `PlantUML <https://plantuml.com/>`__ and track them like any other text file. If you need large files, such as computer-aided design (CAD) files, you might want to look into `Git Large File Storage <https://git-lfs.com/>`__ to point to the files in a separate location.
```

Comment:
In the second sentence, should "simply" be "simple" instead?

Reply:
```suggestion
Unsurprisingly, the answers on what to do varied based on what was required. For example, if you only need simple diagrams, you can create them using a diagrams-as-code tool such as `Mermaid <https://mermaid.js.org/>`__ or `PlantUML <https://plantuml.com/>`__ and track them like any other text file. If you need large files, such as computer-aided design (CAD) files or videos, you might want to look into `Git Large File Storage <https://git-lfs.com/>`__ to point to the files in a separate location.
```